fendermint/testing/materializer/golden/manifest/json/manifest.json

```diff
@@ -1,99 +1,97 @@
 {
   "accounts": {
-    "AJxB": {},
-    "Aj0I_t": {},
-    "TeBcA": {},
-    "ccXUc3": {}
+    "4rENv3UF": {},
```

Comment:
What is the plan with these golden manifest files, should we not pick one of json/toml? I find the json one much easier to read and understand than the others. Also, if we commit this in the repo, what is the rule for when it changes, I guess its seeded somewher so it should change but didn't find that in the code

Reply:
I'd pick YAML, defnitely not TOML, potentially JSON. Not sure we have to pick one, or let users pick their favourite and just pick the right deserializer based on the extension. We can remove the golden files for formats that nobody would pick.

Unfortunately it's not seeded in a repeatable way (quickcheck doesn't support seeding :( ), so if you delete it will be a new random one. It just serves as 1) illustration and 2) early warning for breaking changes.

Reply:
Would consider options like [HCL](https://github.com/hashicorp/hcl) (which formally translates to JSON), and [JSON5](https://json5.org/), which are good tradeoffs between human and machine readable.